CriticalMass/FormatDisplay.swift

```diff
@@ -31,10 +31,11 @@ private extension DateComponents {
 enum FormatDisplay {
     // inject current date to be able to keep the same time difference between tweet date and current date
     static var currentDate = Date()
+    static var calendar = Calendar.current
```

Comment:
Now that I look at it, I don't think its a good idea declare them statically because it would never change after launching the App right?

Reply:
True. How about having the calendar as a default parameter in the signature?

Reply:
I moved both calendar and currentDate to the argument, but that breaks snapshots testing as they rely on that :/

Reply:
Whoat? Sounds strange as to me this is the same